welcome/index.adoc

```diff
@@ -198,83 +197,76 @@ Develop and deploy containerized applications with {product-title}. {product-tit
 
 - **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the {product-title} web console or OpenShift CLI (`oc`) to organize and share the software you develop.
 
-- **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Work with applications]**:
+- **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Work with applications]**.
 
 - **Use the xref:../web_console/web-console-overview.adoc#about-developer-perspective_web-console-overview[*Developer perspective*]** in the {product-title} web console to
 xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[create and deploy applications].
 
 - **Use the xref:../applications/odc-viewing-application-composition-using-topology-view.adoc#odc-viewing-application-composition-using-topology-view[*Topology*] view**
 to see your applications, monitor status, connect and group components, and modify your code base.
 
-- **xref:../applications/connecting_applications_to_services/understanding-service-binding-operator.adoc#understanding-service-binding-operator[Connect your workloads to backing services]**: The Service Binding Operator enables application developers to easily bind workloads with Operator-managed backing services by automatically collecting and sharing binding data with the workloads. The Service Binding Operator improves the development lifecycle with a consistent and declarative service binding method that prevents discrepancies in cluster environments.
-
-- ** xref:../cli_reference/odo-important-update.adoc#odo-important_update[Use the developer CLI tool (`odo`)] **: 
-The `odo` CLI tool lets developers create single or multi-component applications easily and automates deployment, build, and service route configurations. It abstracts complex Kubernetes and {product-title} concepts, allowing you to focus on developing your applications.
```

Comment:
Writer's note: Removed `odo` index items because they include a link that further links to a third-party application. Confirmed with Robert Kratky, but can request a more authentic response, if desired.